app/controllers/works_controller.rb

```diff
@@ -40,8 +44,11 @@ def show
       end
       service = S3QueryService.new(@work, false)
```

Comment:
@jrgriffiniii this will get the wrong objects for procreation, since it is set to false.  I was thinking we would want to retrieve both, but that might be something for discussion.

Reply:
I am sorry, I see this now, I can fix these errors immediately.